package.json

```diff
@@ -15,7 +15,16 @@
   "dependencies": {
     "commander": "2.9.0",
     "open": "0.0.5",
-    "shelljs": "0.5.3"
+    "shelljs": "0.5.3",
```

Comment:
yes, I was going to start looking at bumping these deps and others in kw-web-suite too